Week1/Exersise2-W1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Exersise2_W1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int i;
+            int j;
+            for (i = 1; i <= 10; i++)
+            {
+                for (j = 1; j <=9; j++) 
+                {
+                    int ji = j * i;
+                    Console.Write($"|{j} x {i} = {ji}\t");
+                }
+            Console.WriteLine();
```

Comment:
Let think in a simple way, you write `Console.WriteLine()` to enter a line. So what do you do if you want 1 character "|" before entering the line?